hq/app/api/Snyk.scala

```diff
@@ -48,18 +44,16 @@ object Snyk {
             "patched" -> "false"
           )
         )
-        wsClient.url(snykProjectUrl)
-          .addHttpHeaders("Authorization" -> s"token ${token.value}")
-          .post(projectIssuesFilter)
-
+        snykRequest(token, snykProjectUrl, wsClient).post(projectIssuesFilter)
       })
     Attempt.traverse(projectVulnerabilityResponses) {
-      projectVulnerabilityResponse =>
-        Attempt.fromFuture(projectVulnerabilityResponse) {
-          case NonFatal(e) =>
-            val failure = Failure(e.getMessage, "Could not read project vulnerabilities from Snyk", 502, None, Some(e))
-            FailedAttempt(failure)
-        }
+      projectVulnerabilityResponse => handleFuture(projectVulnerabilityResponse, "project vulnerabilities")
     }
   }
+
+  def handleFuture[A](future: scala.concurrent.Future[A], label: String)(implicit ec: ExecutionContext) = Attempt.fromFuture(future) {
```

Comment:
The file should have `import scala.concurrent.Future` in it, no need to fully qualify Future on an argument's type annotation.

Reply:
Done.